radish34/api/src/db/models/modules/proposals.js

```diff
@@ -82,11 +82,13 @@ export const getAllProposals = async () => {
 
 export const saveProposal = async doc => {
   try {
+    const { _id } = doc;
```

Comment:
this can be inlined - as was in the original file

Reply:
In order for keeping the code consistent and following destructuring throughout the code, I would suggest keeping destructuring even for a single property. Still, should we change remove destructuring for a single property?

Reply:
Did the changes and pushed the commit.